docker/image_types.yaml

```diff
@@ -3,17 +3,19 @@
 
 'base-image':
   tag: 'base-image'
-  context: 'docker/base-image'
+  dockerfile: 'docker/base-image/Dockerfile'
+  path: 'docker/base-image'
```

Comment:
Why are we changing context to path? The docker terminology is "context": https://docs.docker.com/engine/reference/commandline/build/

Reply:
Building this image would use this command:
`docker build docker/base-image --tag  gcr.io/fuzzbench/base-image --file docker/base-image/Dockerfile`
The best way to describe `docker/base-image` based on the documentation you linked to seems to be path, context is `the set of files located in the specified PATH or URL`. Context doesn't seem like a bad way to describe it based on this, but it seems less strictly correct. Seeing the name context used as a variable in the generation code also was confusing, it doesn't convey "the path which the docker build has access to" as well as "path" imo.

Reply:
I think "path" is much less descriptive than "context". A "path" to what? If we really need to rename this, i'd suggest making it more descriptive instead, e.g. "context_path" or "context_directory". However, I personally don't see a reason to rename it: the docker build "context" is the correct terminology describing the source directory used to build a container, ie. it is well known to be a path or an url to a directory.

Also note that if we rename it, we'll need to update all the other code (rule creation for makefile, job creation for queue)

Reply:
To confirm, should this be reverted back to "context" ?

Reply:
Reverted